uint16 Beckhoff
uint16 Odot
uint16 DeltaControlWord
int32 DeltaTargetPos
int32 DeltaTargetVel
int16 DeltaTargetTor
byte DeltaOpMode
---
